packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -45,8 +45,18 @@ type BookEventFormProps = {
 type DefaultValues = Record<string, unknown>;
 
 export const BookEventForm = ({ onCancel }: BookEventFormProps) => {
+  let uid = "";
```

Comment:
why not a state variable here ?

Reply:
This 'uid' info is very much local to BookEventForm (that too for embed scenario only) and not used in any other Component (also dint forsee future use), that's why did not make it as State variable.

Reply:
you could also use useRef

Reply:
yes useRef can be used, updated the same